tests/test_util.py

```diff
@@ -49,60 +47,6 @@ def test_write_templatized_file_dockefile_default(
         template_dir=template_dir,
     )
 
-
```

Comment:
Is all of this removed code related to the tests that tested unused code?

Reply:
yeah